lib/ash_authentication_phoenix/components/password/sign_in_form.ex

```diff
@@ -213,6 +225,13 @@ defmodule AshAuthentication.Phoenix.Components.Password.SignInForm do
     end
   end
 
+  defp get_auth_path_params(params, user) do
+    case Map.get(params, "remember_me") do
+      nil -> %{token: user.__metadata__.token}
```

Comment:
Looking at this function, I'm realizing that I don't think we would need to generate remember me tokens with the standard sign in action, only on the "sign in with token" action. I remember seeing an example on the other PR where both actions are generating remember me tokens.